google-services-plugin/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2"
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
```

Comment:
Note https://github.com/google/play-services-plugins/pull/33/files#diff-39ae392b6c3f8f52448a47e0b8fdbc0f

Reply:
I'm not sure I follow. Should I revert to `1.6`? The stray comment is part of a separate PR and `1.8.4` is the current latest version.

Reply:
Just an FYI there will be a conflicting change when you try to merge this (or vice versa depending on order).